test/Microsoft.Azure.Devices.Edge.Test.Common/linux/PackageManagement.cs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Linux
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public class PackageManagement
```

Comment:
This class is a great solution for handling linux variations in `EdgeDaemon`. But rather than exposing several public properties it should expose methods that use the (private) properties: `InstallAsync` and `UninstallAsync`. And I guess you would still need one property for the list of services that gets passed to start/stop.

This will make `EdgeDaemon.cs` a lot cleaner.

Reply:
Thank you for the advice! I love the idea. I'm working on it.